nestkernel/nest_names.h

```diff
@@ -259,13 +262,16 @@ extern const Name I_T;
 extern const Name I_ahp;
 extern const Name I_e;
 extern const Name I_h;
+extern const Name I_SIC;
```

Comment:
As discussed, I changed "SIC" to "I_SIC" because it is a current. I still use uppercase letters since it is an abbreviation (slow inward current). Please let me know if there is further suggestion. Thanks!